Bottom: 04220695b786dde9de1a9a8a99b42ee430fc83d7
Top:    454e790e6e530719bcb4f3b929de6890768bdf66
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-11-06 02:29:02 +0000

Refresh of important-dont-draw-pour-outli

---

diff --git a/src/draw.c b/src/draw.c
index f6c714b..8ad94c5 100644
--- a/src/draw.c
+++ b/src/draw.c
@@ -708,7 +708,8 @@ pour_callback (const BoxType * b, void *cl)
   struct pin_info *i = (struct pin_info *) cl;
   PourType *pour = (PourType *)b;
 
-  DrawPour (i->Layer, pour, 0);
+  if (gui->gui)
+    DrawPour (i->Layer, pour, 0);
 
   if (pour->PolygonN)
     {
